ApiQuotaHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\GoogleAnalyticsImporter;
+
+/**
+ * Utility class with methods to manage Google Analytics Importer INI configuration.
+ */
+class ApiQuotaHelper
+{
+    /**
+     * Calculate the daily maximum api quota available for the instance
+     * @return int
+     */
+    public function getMaxDailyApiQuota($setTrial = false)
+    {
+        //Local installation. No limitations from google applicable
+        return -1;
+    }
+
+    public function getBalanceApiQuota()
+    {
+        return $this->getMaxDailyApiQuota();
+    }
+
+    public static function saveApiUsed($numQueries)
+    {
+        //No need to save since it's a local installation
+        return;
+    }
+
+    public static function trackEvent($event, $name)
+    {
+        //DI to takeover on ConnectAccount
+        return;
+    }
+}
```

Comment:
It kind of seems like this class and [the one from ConnectAccounts](https://github.com/innocraft/plugin-ConnectAccounts/blob/feature/PG-1351-manage-internal-quota/ApiQuotaHelper.php) should share an interface or abstract parent?

Reply:
Updated to extend